video/transmux.go

```diff
@@ -81,7 +82,7 @@ func MuxTStoFMP4(fmp4ManifestOutputFile string, inputs ...string) error {
 
 	timeout, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
 	defer cancel()
-	cmd := exec.CommandContext(timeout, "ffmpeg", args...)
+	cmd := exec.CommandContext(timeout, "/path/to/ffmpeg-nice.sh", args...)
```

Comment:
This is pseudo-code. I'm not even sure if these two places are the correct ones for VOD processing

Reply:
This is just the MP4 generation step. There's ffmpeg usage in different places for for segmenting, thumbnailing, clipping, etc. Just grep-ing `ffmpeg` or `CommandContext | grep ffmpeg` should get you most of the instances.

Reply:
So we're not fully sure yet but it looks like some `ffmpeg` for big VODs are congesting both disk writes on datapacket servers and generate big egress spikes resulting in entire server slowdown and k3s restarts. 
[We're testing with Rafał](https://discord.com/channels/423160867534929930/1219450408439648418/1222181474275430451) if moving VOD processing onto just one of the nodes in the cluster results in less or zero restarts of the rest of the nodes.